common/app/model/dotcomrendering/DotcomRenderingUtils.scala

```diff
@@ -30,9 +30,42 @@ import views.support.AffiliateLinksCleaner
 
 import java.net.URLEncoder
 
+sealed trait DotcomRenderingMatchType
+
+object DotcomRenderingMatchType {
+  implicit val matchTypesWrites = new Writes[DotcomRenderingMatchType] {
+    def writes(matchType: DotcomRenderingMatchType) =
+      JsString(matchType.toString)
+  }
+}
+
+case object CricketMatchType extends DotcomRenderingMatchType
+case object FootballMatchType extends DotcomRenderingMatchType
+
+case class DotcomRenderingMatchData(matchUrl: String, matchType: DotcomRenderingMatchType)
+
 object DotcomRenderingUtils {
+  def makeMatchData(articlePage: ContentPage): Option[DotcomRenderingMatchData] = {
+    makeFootballMatch(articlePage).orElse(makeCricketMatch(articlePage))
+  }
+
+  def makeCricketMatch(articlePage: ContentPage): Option[DotcomRenderingMatchData] = {
+    val cricketDate = articlePage.getJavascriptConfig.get("cricketMatchDate").map(_.toString)
+    val cricketTeam = articlePage.getJavascriptConfig.get("cricketTeam").map(_.toString)
+
+    (cricketDate, cricketTeam) match {
+      case (Some(date), Some(team)) =>
+        Some(
+          DotcomRenderingMatchData(
+            s"${Configuration.ajax.url}/sport/cricket/match/${date.replaceAll("\"", "")}/${team.replaceAll("\"", "")}.json?dcr=true",
```

Comment:
Since `articlePage.getJavascriptConfig.get("xxx")` is already a JSValue, it wrapped both the values in double quotes, e.g the content might be `"england-cricket-team"`, we need to remove this from the string to put it in the URL

If anyone knows a better way I'd love to hear it, as I'm not a massive fan of this one!